test/SpokePool.RelayerRefund.ts

```diff
@@ -0,0 +1,22 @@
+import { expect } from "chai";
```

Comment:
In your next PR it might be good to re-name this file to `SpokePool.RefundInitialization.ts` and then you can create `SpokePool.RefundExecution.ts` for the execution step, following the pattern I did in the hub.

Reply:
done!